src/webkit/wkBrowser.ts

```diff
@@ -137,19 +129,25 @@ export class WKBrowser extends platform.EventEmitter implements Browser {
 
   _onPageProxyDestroyed(event: Protocol.Browser.pageProxyDestroyedPayload) {
     const pageProxyId = event.pageProxyId;
-    const pageProxy = this._pageProxies.get(pageProxyId)!;
+    const pageProxy = this._pageProxies.get(pageProxyId);
+    if (!pageProxy)
```

Comment:
How can this happen?

Reply:
If the proxy belongs to default context, and `!this._attachToDefaultContext`.